### ISSUE TITLE
Handle SQLException "Invalid timestamp from server: null" for "null" value

### ISSUE THREAD

**Issue by nagarockset** [rockset/rockset-java-client#10]
When you join collections with timestamp casted columns having null values, driver is throwing SQLException "Invalid timestamp from server: null" . This is already handled in driver for DATETIME column type , but not for timestamp column type. 

P.S: This error was encountered in executing one of the test case of TDVT.
<img width="960" alt="join null bool" src="https://user-images.githubusercontent.com/78889317/110112912-30dfaa00-7d67-11eb-806d-a3542bf0af74.png">
The console is rightly showing the right values and not throwing any errors.

**Comment by nagarockset** [comment]
Fixed.

**Comment by escapedcanadian** [comment]
I got this same error when trying to open a collection in Tableau. At this point, it shouldn't (yet) be attempting any joins because it is the only collection in the data set. I tried with another collection that should have had a similar structure and didn't get this error. Still trying to figure out what is different between the two.  There are, however, definitely null timestamps in both collections.

![Screen Shot 2021-05-23 at 10 17 51 AM](https://user-images.githubusercontent.com/4163628/119270184-5931b480-bbb0-11eb-9fa5-165ca2b9eaf5.png)
